src/main/resources/io/openliberty/tools/intellij/lsp4jakarta/messages/messages.properties

```diff
@@ -62,20 +62,27 @@ ConstraintAnnotationsMethod = Constraint annotations are not allowed on static m
 ConstraintAnnotationsField = Constraint annotations are not allowed on static fields.
 AnnotationBooleanMethods = The {0} annotation can only be used on boolean and Boolean type methods.
 AnnotationBooleanFields = The {0} annotation can only be used on boolean and Boolean type fields.
+AnnotationBooleanParams = The {0} annotation can only be used on boolean and Boolean type Parameters.
```

Comment:
In this context it looks strange that "Parameters" is capitalized. Please make it lower case. This applies to all the new messages and in some cases twice.